src/Tooltip/index.tsx

```diff
@@ -21,3 +21,19 @@ export function DateWithToolTip(props: { date: Date }) {
     </Tooltip>
   );
 }
+
+export function TextWithToolTipIfTrimmed(props: {
+  text: string;
+  maxChars: number;
+}) {
+  if (props.text.length > props.maxChars) {
+    const trimmedSuffix = `..`;
+    return (
+      <Tooltip title={props.text}>
+        {props.text.substring(0, props.maxChars - trimmedSuffix.length)}
+        {trimmedSuffix}
+      </Tooltip>
+    );
+  }
+  return <span>{props.text}</span>;
```

Comment:
```suggestion
  return props.text;
```

Reply:
Can not be a string.
![image](https://user-images.githubusercontent.com/26680884/173828588-e28bee51-58ba-4691-9422-43157e714ccf.png)

Reply:
```suggestion
  return <>{props.text}</>;
```

Ah right, in that case a fragment can be used.

Reply:
Unfortunately, fragments are not allowed to only contain one child. I remember that we had this discussion before are we aggreed on `<span/>` as the best solution

Reply:
I could als use ´<Typography.Text>´ if we want to keep it in antd style...